src/draw/lv_img_decoder.h

```diff
@@ -41,6 +41,7 @@ typedef enum {
     LV_IMG_DEC_CACHED       = 0x08, /**!< The complete image can be cached (used for rotation and zoom) */
     LV_IMG_DEC_VFR          = 0x10, /**!< The animation has a variable frame rate */
     LV_IMG_DEC_LOOPING      = 0x20, /**!< The animation is looping */
+    LV_IMG_DEC_TRANSPARENT  = 0x40, /**!< The image might have transparent area */
```

Comment:
What about `LV_IMG_DEC_HAS_ALPHA_CHANNEL`?

Reply:
None of the item use sentences above. 

Also, in case of expensive decoding, this flag is set without checking the actual content (hence the "might have"). 
For example, the flag will be set even if a rlottie picture completely cover the rendering area, I'm not checking if any pixels has a non opaque (alpha) value (and it might change during animation).